inductiva/fluids/dam_break/dam_break.py

```diff
@@ -61,16 +61,20 @@ def simulate(
             simulation_time: Simulation time, in seconds.
             run_async: Whether to run the simulation asynchronously.
         """
+        simulator.override_api_method_prefix("dam_break")
 
-        particle_radius = ParticleRadius[resolution.upper()].value
+        self.particle_radius = ParticleRadius[resolution.upper()].value
+        self.simulation_time = simulation_time
+        self.adaptive_time_step = True
+        self.particle_sorting = True
+        self.time_step = 0.001
+        self.output_time_step = 1 / 60
 
-        task = super().simulate(simulator=simulator,
-                                machine_group=machine_group,
-                                device=device,
-                                particle_radius=particle_radius,
-                                simulation_time=simulation_time,
-                                run_async=run_async)
-
-        task.set_output_class(fluids.SPHSimulationOutput)
+        task = super(fluids.FluidBlock, self).simulate(
```

Comment:
This a weird but cool feature, that let's you inherit the `simulate` of the grandparent instead of the parent. That is why `fluids.FluidBlock` is set first -> it says that you will do the super for the FluidBlock, which is `Scenario.simulate()`.

The problem this addresses is that the FluidBlock would override the api_method_name for the DamBreak, hence not allowing us to identify the DamBreak at all. Since I still want to use the file configuration functions, I thought about this method.